shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -62,12 +58,20 @@
  *   <li>{@link #getInitialRoute()}</li>
  * </ul>
  * <p>
+ * The Dart entrypoint and app bundle path are not supported as {@code Intent} parameters due to
```

Comment:
I wouldn't write this down

Reply:
Isn't it important that developers know this? What if they subclass FlutterActivity, add this as an Intent parameter, and then export that Activity?

Reply:
Oh ok, that's a valid concern. Maybe don't make it so ominous? Rather than saying arbitrary Dart code, say arbitrary entrypoint or something.

Reply:
Sure, done.